modules/backend/assets/js/vendor/jquery-migrate.min.js

```diff
@@ -0,0 +1,215 @@
+/*! jQuery Migrate v3.0.1 | (c) jQuery Foundation and other contributors | jquery.org/license */
```

Comment:
Are you sure this is the minified version of jquery-migrate?

Reply:
yes this is the minified version of migrate: https://code.jquery.com/jquery-migrate-3.0.1.min.js